packages/salesforcedx-vscode-core/src/commands/util/index.ts

```diff
@@ -40,7 +40,4 @@ export {
   DevUsernameChecker,
   EmptyPreChecker
 } from './preconditionCheckers';
-export {
-  createComponentCount,
-  useBetaDeployRetrieve
-} from './betaDeployRetrieve';
+export { createComponentCount } from './betaDeployRetrieve';
```

Comment:
We should also export `formatException` here so we can import it elsewhere by just using the `./utils` path